targets/targets.json

```diff
@@ -4007,5 +4007,35 @@
         "detect_code": ["7013"],
         "release_versions": ["5"],
         "bootloader_supported": true
+    },
+    "XMC_XXXX": {
+        "inherits": ["Target"],
+        "extra_labels": ["Infineon", "XMCLib"],
+        "supported_toolchains": ["GCC_ARM", "ARM"],
```

Comment:
No support for IAR? if yes, you may want to add linker scripts for IAR as well.
@0xc0170 : Is IAR support mandatory or optional?

Reply:
IAR support is mandatory, as without it tests and export builds will fail with IAR.

Reply:
@mbed-Infineon-XMC ..Could you please add IAR support as well. You will need to add the linker scripts too under ../device/TOOLCHAIN_IAR/

Reply:
ARM compiler also not supported yet!
I will add support for IAR and ARM in the next days.
Should the Exporter & build test not fail if there is no support for all 3 toolchains?

Reply:
@mbed-Infineon-XMC : The Mbed Enabled requirement is to have support for all 3 toolchains. The CI systems also test the targets against these 3 toolchains. So, if any 1/3 is not present, it will fail :-)